specification/network/resource-manager/readme.java.md

```diff
@@ -20,6 +20,9 @@ directive:
   - rename-operation:
       from: VirtualHubBgpConnection_Delete
       to: VirtualHubBgpConnections_Delete
+  - rename-operation:
+      from: InboundSecurityRule_CreateOrUpdate
```

Comment:
Why do we need this one? Duplicate operation ID?

Reply:
It's required for autorest.java. For operations, the code generator will take `InboundSecurityRule` and prepare keywords of `InboundSecurityRule` and `InboundSecurityRules` while it already exists `InboundSecurityRules` in the swagger definition. Once we rename it to a new operation name, the conflict gets resolved.